deploy/008-configure-acm-commands-aggregator.ts

```diff
@@ -0,0 +1,1101 @@
+      };
+    }),
+    ...accounts.map(timelock => {
+      return {
+        params: [xvsBridgeAdmin, "setWhitelist(address,bool)", timelock],
+      };
+    }),
+    ...accounts.map(timelock => {
+      return {
+        params: [xvsBridgeAdmin, "setConfig(uint16,uint16,uint256,bytes)", timelock],
+      };
+    }),
+    {
+      params: [xvsBridgeAdmin, "sweepToken(address,address,uint256)", AccountType.NORMAL_TIMELOCK],
+    },
+    ...accounts.map(timelock => {
+      return {
+        params: [xvsBridgeAdmin, "updateSendAndCallEnabled(bool)", timelock],
+      };
+    }),
+    {
+      params: [xvsBridgeAdmin, "setTrustedRemoteAddress(uint16,bytes)", AccountType.NORMAL_TIMELOCK],
+    },
+    {
+      params: [xvsBridgeAdmin, "transferBridgeOwnership(address)", AccountType.NORMAL_TIMELOCK],
+    },
+  ];
+};
+
+const getXVSVaultPermissions = (xvsVault: string): Permission[] => {
+  return [
+    {
+      params: [xvsVault, "pause()", AccountType.CRITICAL_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "resume()", AccountType.CRITICAL_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "setRewardAmountPerBlockOrSecond(address,uint256)", AccountType.CRITICAL_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "pause()", AccountType.FAST_TRACK_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "resume()", AccountType.FAST_TRACK_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "setRewardAmountPerBlockOrSecond(address,uint256)", AccountType.FAST_TRACK_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "pause()", AccountType.NORMAL_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "resume()", AccountType.NORMAL_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "add(address,uint256,address,uint256,uint256)", AccountType.NORMAL_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "set(address,uint256,uint256)", AccountType.NORMAL_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "setRewardAmountPerBlockOrSecond(address,uint256)", AccountType.NORMAL_TIMELOCK],
+    },
+    {
+      params: [xvsVault, "setWithdrawalLockingPeriod(address,uint256,uint256)", AccountType.NORMAL_TIMELOCK],
+    },
+  ];
+};
+
+const getPoolRegistryPermissions = (poolRegistry: string): Permission[] => {
+  return [
+    {
+      params: [poolRegistry, "addPool(string,address,uint256,uint256,uint256)", AccountType.NORMAL_TIMELOCK],
+    },
+    {
+      params: [poolRegistry, "addMarket(AddMarketInput)", AccountType.NORMAL_TIMELOCK],
+    },
+    {
+      params: [poolRegistry, "setPoolName(address,string)", AccountType.NORMAL_TIMELOCK],
+    },
+    {
+      params: [poolRegistry, "updatePoolMetadata(address,VenusPoolMetaData)", AccountType.NORMAL_TIMELOCK],
+    },
+  ];
+};
+
+const getPrimePermissions = (prime: string): Permission[] => {
+  return [
+    ...accounts.map(timelock => {
+      return {
+        params: [prime, "updateAlpha(uint128,uint128)", timelock],
+      };
+    }),
+    ...accounts.map(timelock => {
```

Comment:
I'm not sure if we need an inner object, with the `params` field. I think each entry could be a simple array. Moreover, I think we can simplify these `map` calls with `flatMap`:

```
return accounts.flatMap(timelock => {
      return [
            [prime, "updateAlpha(uint128,uint128)", timelock],
            [prime, "setStakedAt(address[],uint256[])", timelock],
            [prime, "addMarket(address,address,uint256,uint256)", timelock],
            // ... and so on
      ];
    });
```